src/main/java/reporting_system/ReportedUser.java

```diff
@@ -0,0 +1,42 @@
+package reporting_system;
+
+import data_access_storage.RequestModel;
+import data_access_storage.UserRepo;
+import data_access_storage.UserRepoManager;
+import entities.UserAccount;
+
+import javax.swing.*;
+import java.io.File;
+import java.io.IOException;
+import java.util.Map;
+
```

Comment:
Similar to BlockedUser.java, a comment for this class would be helpful for readers to understand what it does.